dev/com.ibm.ws.jaxrs.2.0.cdi.1.2_fat/.classpath

```diff
@@ -4,14 +4,16 @@
 	<classpathentry kind="src" path="test-applications/basic/src"/>
 	<classpathentry kind="src" path="test-applications/beanvalidation/src"/>
 	<classpathentry kind="src" path="test-applications/complex/src"/>
-        <classpathentry kind="src" path="test-applications/contextandCDI/src"/>
+	<classpathentry kind="src" path="test-applications/contextandCDI/src"/>
 	<classpathentry kind="src" path="test-applications/contextandclient/src"/>
 	<classpathentry kind="src" path="test-applications/disable/src"/>
 	<classpathentry kind="src" path="test-applications/lifecyclemethod/src"/>
 	<classpathentry kind="src" path="test-applications/lifecyclemismatch/src"/>
 	<classpathentry kind="src" path="test-applications/loadonstartup/src"/>
 	<classpathentry kind="src" path="test-applications/resourceInfoAtStartup/src"/>
 	<classpathentry kind="src" path="test-applications/interceptor/src"/>
+	<classpathentry kind="src" path="test-applications/injectIntoApp/src"/>
+	<classpathentry kind="src" path="test-applications/cdiinjectintoapp/src"/>
```

Comment:
minor nit, but can you move these up to where they belong in alphabetical order so that the test apps will be easier to find in Eclipse?